go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/mezo-org/mezod
 
-go 1.22.2
+go 1.22.11
 
-toolchain go1.22.5
+toolchain go1.23.4
```

Comment:
Why 1.23?

Reply:
It just updated by itself with the `go get github.com/cometbft/cometbft@v0.38.17` to be honest, let me check and change it manually.